cli/inspector.rs

```diff
@@ -547,6 +538,21 @@ impl DenoInspector {
       };
     }
   }
+
+  /// This function blocks the thread until at least one inspector client has
+  /// established a websocket connection and successfully completed the
+  // handshake. After that, it instructs V8 to pause at the next statement.
+  pub fn wait_for_session_and_break_on_next_statement(&mut self) {
+    loop {
+      match self.sessions.get_mut().established.iter_mut().next() {
+        Some(session) => break session.break_on_next_statement(),
+        None => {
+          self.flags.get_mut().waiting_for_session = true;
+          let _ = self.poll_sessions(None).unwrap();
+        }
+      };
+    }
+  }
```

Comment:
Can we turn this method into an `async` one?

Reply:
I guess, but what's the point?
I also don't think we can await inside deno_mod_evaluate but not sure...

Reply:
I'm not against it btw

Reply:
Just sounds like $work where there is no need for it

Reply:
> I guess, but what's the point?

It's a hot loop isn't it? Won't it cause high CPU usage when waiting for debugger? 

> I also don't think we can await inside deno_mod_evaluate but not sure...

IMO it should be called in `Worker::execute_module` (and `Worker::execute_module_from_code`) because Inspector is CLI-only concept right now.

Reply:
> I'm not against it btw
> Just sounds like $work where there is no need for it

I'm happy to clean this up later

Reply:
> It's a hot loop isn't it? Won't it cause high CPU usage when waiting for debugger?

poll_sessions() puts the thread to sleeps until there is a connected session (when the `wait_for_session` flag is set)